src/utils.js

```diff
@@ -190,3 +190,26 @@ export const NewsUtil = {
 		];
 	},
 };
+
+export const AdminlyUtil = {
+	getAdminlyColor: (string) => {
+		const adminlyColors = [
+			"#8AE21A",
+			"#E21A7A",
+			"#D21AE2",
+			"#5A1AE2",
+			"#1A76E2",
+			"#1AB2E2",
+			"#1AE2D6",
+			"#1AE2A6",
+			"#010564",
+			"#F68500",
```

Comment:
Interesting solution but should we not have this specified as css variables instead.

I suggest a `global.css` in adminly_core

with 
```
:root {
   --adminly-color-1: #8AE21A; 
   --adminly-color-2: #E21A7A; 
   ...

}
```

and then the util function is just applying random css class.